server/services/employers.js

```diff
@@ -34,13 +34,17 @@ const saveSites = async (sitesArg) => {
   return response;
 };
 
+const updateSite = async (id, site) => dbClient.db[collections.EMPLOYER_SITES]
```

Comment:
Think this will require a little more work. As pointed out in the ticket, we'll need to add to the history object similar to what we're doing on other PATCH endpoints.